### ISSUE TITLE
Blurry text short after scrolling

### ISSUE THREAD

**Issue by hsandt** [ricksladkey/SmoothScroll#12]
The text is blurry a few tenths of second after scrolling finished. Then the text is synced to screen pixels and becomes sharp again.

This happens for all Animation speed values (Slow, Normal, Fast). I have the impression that because Slow scrolling is longer, the blur time will be shorter, so maybe the time before text becomes sharp again is actually fixed.

Expected behavior: as soon scrolling finishes, the text should become sharp again. This means that the time before it sync with screen pixels shouldn't be fixed, but depend on scrolling speed.

Just after scroll:
![smoothscroll blurry text just after scroll 1](https://user-images.githubusercontent.com/1067772/40354364-facaa51c-5db3-11e8-88ed-6d7c93b2c02e.png)

After waiting a few tenths of second:
![smoothscroll blurry text just after scroll 2](https://user-images.githubusercontent.com/1067772/40354365-fae61ff4-5db3-11e8-86d9-add1a07e0488.png)

**Comment by ricksladkey** [comment]
Interesting. I do not experience this symptom! Visual Studio is the one doing the scrolling, so my guess is that you are seen artifacts of the graphical rendering peculiar to your video driver.

The only thing I can suggest to make this symptom go away for you is to try (at least as an experiment) to disable graphics hardware acceleration in Tools -> Options:

![image](https://user-images.githubusercontent.com/2680524/40398951-ab3fb6b8-5e08-11e8-8710-472ccfeb67a2.png)

**Comment by hsandt** [comment]
Hm... oops, it seems like I was using the _other_ [SmoothScroll plugin](https://github.com/lkytal/SmoothScroll) by Kaiyuan Liu.
But your advice was generic so I tried it anyway, but it just delayed the blur effect. I have to post an issue of the other plugin's GitHub now.
I also tried your plugin since you mentioned it doesn't have the blur issue at all, but it doesn't seem to work with my mouse (maybe the events are aggregated as mentioned in https://github.com/ricksladkey/SmoothScroll/issues/3, but I don't manage to use Spy to check them for some reason). I guess I'll go without smooth scrolling for now.
